### PR TITLE
Tests/adapting tests to validate ts iso 8601

### PR DESCRIPTION
**Pull request by manrodrigues**
- avoiding Expecting value: line 1 column 1 (char 0) error
- adapting tests to validate ts iso 8601

### CODE REVIEW

File: python-test/features/steps/control_plane_policies.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from random import choice, choices, sample
 from deepdiff import DeepDiff
 import json
+import ciso8601
```
Comment (lpegoraro) [inline python-test/features/steps/control_plane_policies.py:13]: 😍

File: python-test/features/steps/control_plane_policies.py [hunk 2]
```diff
@@ -643,7 +655,10 @@ def is_expected_msg_in_log_line(log_line, expected_message, list_agent_policies_
     if log_line is not None:
         if expected_message in log_line['msg'] and 'policy_id' in log_line.keys():
             if log_line['policy_id'] in list_agent_policies_id:
-                if log_line['ts'] > considered_timestamp:
+                if isinstance(log_line['ts'], int) and log_line['ts'] > considered_timestamp:
+                    return True
+                elif isinstance(log_line['ts'], str) and datetime.timestamp(ciso8601.parse_datetime(log_line['ts'])) > \
+                        considered_timestamp:
```
Comment (lpegoraro) [inline python-test/features/steps/control_plane_policies.py:661]: nice to have standards right?

**Review by lpegoraro — APPROVED** [review]
LGTM

**Review by etaques — APPROVED** [review]
LGTM

**Pull request merged**